watch-and-show-snapshots.js

```diff
@@ -1,16 +1,19 @@
 let fs = require('fs')
+let path = require('path')
 let chokidar = require('chokidar')
 let parseGitignore = require('parse-gitignore')
 
 let updateAndShowSnaphots = require('./update-and-show-snapshots')
 
+let rootPath = path.dirname(require.main.filename)
```

Comment:
`require.main.filename` will point to `/path/to/the/project/node_modules/.bin/print-snapshots`.

I think we need [`pkg-up`](https://www.npmjs.com/package/pkg-up) to find `package.json` and be sure about the root of the project.

Reply:
It seems, I need to sleep a little :-)